From 6949c2591cd34835aaf425c87051a77a828ccad2 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Tue, 6 Dec 2011 14:01:16 +0800
Subject: [PATCH 0692/2463] ENGR00169509-1 ipuv3 fb: change wait for vsync
 ioctl irq from eof to nfack

change wait for vsync ioctl irq from eof to nfack

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 arch/arm/plat-mxc/include/mach/ipu-v3.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/ipu-v3.h b/arch/arm/plat-mxc/include/mach/ipu-v3.h
index dd18f36..a38b465 100755
--- a/arch/arm/plat-mxc/include/mach/ipu-v3.h
+++ b/arch/arm/plat-mxc/include/mach/ipu-v3.h
@@ -342,6 +342,10 @@ enum ipu_irq_line {
 	IPU_IRQ_BG_ALPHA_SYNC_EOF = 51,
 	IPU_IRQ_BG_ALPHA_ASYNC_EOF = 52,
 
+	IPU_IRQ_BG_SYNC_NFACK = 64 + 23,
+	IPU_IRQ_FG_SYNC_NFACK = 64 + 27,
+	IPU_IRQ_DC_SYNC_NFACK = 64 + 28,
+
 	IPU_IRQ_DP_SF_START = 448 + 2,
 	IPU_IRQ_DP_SF_END = 448 + 3,
 	IPU_IRQ_BG_SF_END = IPU_IRQ_DP_SF_END,
-- 
1.7.7.4

